spec/features/admin_invoice_settings_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+RSpec.feature "Print Invoice Settings", js: true do
+  given!(:order) { FactoryGirl.create(:completed_order_with_totals) }
+  given(:user) { FactoryGirl.create(:admin_user, password: "boxen1011") }
+
+  scenario "It allows for the update of print invoice settings" do
+    visit spree.admin_path
+
+    fill_in "Email", with: user.email
+    fill_in "Password", with: "boxen1011"
+    click_button "Login"
+
+    click_link "Configuration"
+
+    select "Times Roman", from: "print_invoice_font_face"
+    fill_in "print_invoice_next_number", with: "150"
```

Comment:
I'd probably use the words of the label instead of the name of the field to target the input

```rb
fill_in("Next Invoice Number", with: "150")
```